WebContent/resources/common.js

```diff
@@ -1533,7 +1533,7 @@ function ReportPointsContext(startContext, points) {
        for (let i = 0; i < this.contextArray.length; i++) {
           context[context.length] = {
                 pointId: this.contextArray[i].pointId,
-                pointXid: this.contextArray[i].xid,
+                pointXid: this.contextArray[i].pointXid,
```

Comment:
This needs to be reverted and corrected in a separate issue, with a description of the bug, it seems that it should be included in version 2.8.0